src/index.ts

```diff
@@ -0,0 +1,4 @@
+export { default as Button } from "./8bit/button";

```

Comment:
@andrewgierens the idea with this file was to allow us to export different "UIs"